### PR TITLE
Added ability to write to metadata directly from bqInsertData

### PR DESCRIPTION
**Pull request by byapparov**
…rtData function

### CODE REVIEW

**Comment by byapparov** [comment]
Example of refactoring impact in [pull request](https://github.com/madedotcom/customer-analytics/pull/289)

File: tests/testthat/test.bq.query.R [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  query_function <- mock(cycle = T)
+  with_mock(
+    `bigrquery::query_exec` = query_function,
+    {
+      res <- bqExecuteSql("SELECT %1$s", "TEST")
+      expect <- "SELECT TEST"
+      expect_args(query_function, 1,  sql = expect, project = "", default_dataset = "", max_pages = Inf)
+
+      res <- bqExecuteFile("sample_query.sql", 100)
+      expect <- "SELECT TEST_100"
```
Comment (lmavridis) [inline tests/testthat/test.bq.query.R]: I don't see how expect is checking the result of res

File: tests/testthat/test.bq.query.R [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+library(mockery)
+context("BigQuery query functions.")
+
+test_that("Correct sql is executed", {
+  query_function <- mock(cycle = T)
+  with_mock(
+    `bigrquery::query_exec` = query_function,
+    {
+      res <- bqExecuteSql("SELECT %1$s", "TEST")
+      expect <- "SELECT TEST"
```
Comment (lmavridis) [inline tests/testthat/test.bq.query.R]: I don't see how expect is checking the result of res

File: tests/testthat/test.bq.query.R (hunk 1 above)
Comment (byapparov) [inline tests/testthat/test.bq.query.R]: line bellow

**Pull request merged**